### ISSUE TITLE
Change 'Read the version currently in force' to 'Read the latest available version'?

### ISSUE THREAD

**Issue by goose-life** [laws-africa/peachjam#1495]
e.g. https://lawlibrary.org.za/akn/za/act/1959/58/eng@1959-07-03 has 'This is the version of this Act as it was from 3 July 1959 to 14 March 1974. Read the version currently in force.'
<img width="761" alt="image" src="https://github.com/laws-africa/peachjam/assets/32566441/c4b9de29-4000-40b6-a313-1af8ed060055">

If you click through, you get to https://lawlibrary.org.za/akn/za/act/1959/58/eng@1977-04-27, which has 'There are outstanding amendments that have not yet been applied. See the History tab for more information.'
<img width="798" alt="image" src="https://github.com/laws-africa/peachjam/assets/32566441/1f58672e-1efc-43fe-bd46-f367c0eabfdf">

In other words, older points in time shouldn't assume that the link to the latest point in time is the version currently in force. All they 'know' is that that link is to the latest available version.
